modules/statistics/AvgRTPStatsReporter.js

```diff
@@ -223,26 +219,33 @@ class ConnectionAvgStats {
             return;
         }
 
+        if (!data.transport || !data.transport.length) {
```

Comment:
in what case there is no transport ? I wonder if the stats should not be cleared in such case, otherwise the avg may not be accurate (if there's no transport for too long)